_data/simple-icons.json

```
@@ -6454,6 +6454,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
+            "hex": "000000",
```

```suggestion
            "hex": "1E1E1E",
```

Header uses the above hex code, we should stick with that.